components/configurator/ui/ConfiguratorScreen.js

```diff
@@ -17,6 +17,7 @@ const Frame = styled.div`
 `
 
 const Screen = styled.div`
+  position: relative;
```

Comment:
I don't think this breaks other uses of the `ConfiguratorScreen` - any concerns with this?

Reply:
I can’t think of any case where something in the screen needs to expand beyond it, so 👍